lib/yardstick/v2_client/booking.rb

```diff
@@ -0,0 +1,20 @@
+require 'yardstick/v2_client/remote_model'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.